### ISSUE TITLE
The classifier_free guidance in the training process

### ISSUE THREAD

**Issue by liupengcnu** [levihsu/OOTDiffusion#122]
Hi, Dr Xu, the classifier_free guidance in training process is same to your published inference code, as shown in the following code

# for prompt_embeds
prompt_embeds = torch.cat([prompt_embeds, prompt_embeds])

# for garm_latented
uncond_image_latents = torch.zeros_like(image_latents)
image_latents = torch.cat([image_latents, uncond_image_latents], dim=0)

# for vton_latented
image_latents = torch.cat([image_latents] * 2, dim=0)

# for outfitting dropout
if random.uniform(0, 1) < 0.1:
       garm_latents = torch.zeros(garm_latents.shape)
